src/FACT_internal.c

```diff
@@ -676,6 +678,33 @@ uint8_t FACT_INTERNAL_CreateSound(FACTCue *cue, uint16_t fadeInMS)
 			}
 		}
 
+		/* Calculate Max RPC Release Time */
```

Comment:
Does this still work if you fold the `i` loop into the one right above it? That way we're only looping through the track list once but still doing the ReleaseTime check.

Reply:
Yeah totally.  Last night I was going for getting the feature to work right and pass my test. :)